src/axel_f/excel/coerce.cljc

```diff
@@ -27,3 +28,62 @@
     true "TRUE"
     false "FALSE"
     (str item)))
+
+(defn to-string*
+  "Try to coerce given value to a string type. Returns null for empty value."
+  [^{:doc "Any object to coerce to a string."} obj]
+  (when obj (str obj)))
+
+(def to-string #'to-string*)
+
+(defn to-integer*
+  "Try to coerce given value to an integer type. Returns null for empty or not reducible to an integer type value."
```

Comment:
`Returns null for empty or not reducible to an integer type value.` is a bit confusing. Is 'not reducible' the error response for an integer type? Or is there a general error 'Not reducible to an integer type'? 

'Tries to coerce given value to an integer type. '

Reply:
I wanted to outline that null will be a result of `to-integer('foo')`. Underling function will throw an exception but the function catches and transforms it to null.